docker/iroh-fedimintd/docker-compose.yaml

```diff
@@ -10,6 +10,8 @@ services:
       - FM_BITCOIN_NETWORK=signet
       - FM_BITCOIN_RPC_KIND=esplora
       - FM_BITCOIN_RPC_URL=https://mutinynet.com/api
+      - FM_BIND_P2P=0.0.0.0:8173
+      - FM_BIND_API=0.0.0.0:8174
       - FM_BIND_UI=0.0.0.0:8175
```

Comment:
TODO: we should bind this to docker network only, as we don't want to expose it publically, I think.